.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           ruby-version: ${{ matrix.ruby-version }}
           bundler-cache: true
       - name: Set up Google Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: google-github-actions/setup-gcloud@v0
```

Comment:
why not v1? https://github.com/google-github-actions/setup-gcloud#service-account-key-json